video/agon_screen.h

```diff
@@ -106,6 +106,12 @@ int change_resolution(int colours, char * modeLine) {
 	controller->enableBackgroundPrimitiveTimeout(false);
 
 	canvas = new fabgl::Canvas(controller);			// Create the new canvas
+	debug_log("after change of canvas...\n\r");
+	debug_log("  free internal: %d\n\r  free 8bit: %d\n\r  free 32bit: %d\n\r",
+		heap_caps_get_free_size(MALLOC_CAP_INTERNAL),
+		heap_caps_get_free_size(MALLOC_CAP_8BIT),
+		heap_caps_get_free_size(MALLOC_CAP_32BIT)
+	);
```

Comment:
oops - didn't really intend to commit this.  added to aid in investigations as to why some screen modes are problematic.  however it's inclusion is relatively harmless so will leave